styleSheet.css

```diff
@@ -236,6 +236,58 @@ ul {
   width: 100%;
 }
 
+#awardsInterest {
+  background-color: var(--dark__primary);
+  font-size: 1.2em;
+  height: calc(60vh + 20vw);
+  display: inline-flex;
+}
+
+.awardsInterest__list {
+  background-color: var(--light__primary);
+  border-radius: 20px;
+  height: 90%;
+  margin: 2%;
+  padding: 0 10px;
+  position: initial;
+  width: 85%;
+  z-index: 20;
+}
+
+.awardsInterest__list > h3 {
+  font-size: calc(2rem + 0.5vw);
+  margin-bottom: 0%;
+  text-align: center;
+}
+.awardsInterest__list > p {
+  margin-top: 0%;
+  text-align: center;
+}
+
+.awardsInterest__list > ul {
+  font-size: calc(1rem + 1vw);
+  text-shadow: 1px 1px var(--background__color);
+}
+
+.interest {
+  margin-left: 0;
+}
+
+.interest>ul li:nth-child(odd){
+  display: inline;
+  float: left;
+  margin: 0 calc(15% + 2vw);
+}
+
+.flip-vertical-right {
+	-webkit-animation: flip-vertical-right 0.4s cubic-bezier(0.455, 0.030, 0.515, 0.955) both;
+	        animation: flip-vertical-right 0.4s cubic-bezier(0.455, 0.030, 0.515, 0.955) both;
+}
+
+.flip-vertical-left {
+	-webkit-animation: flip-vertical-left 0.4s cubic-bezier(0.455, 0.030, 0.515, 0.955) both;
```

Comment:
You could have also a var for the time `0.4s` this way if you change the var it will be changed in all the places

Reply:
Oooh, yeah, I'll do so